src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewerUtils.java

```diff
@@ -52,10 +52,13 @@
 import java.util.stream.Collectors;
 
 public class ProfileViewerUtils {
-	static final HashMap<String, ItemStack> playerSkullCache = new HashMap<>();
+	static Map<String, ItemStack> playerSkullCache = new HashMap<>();
 
 	public static JsonArray readInventoryInfo(JsonObject profileInfo, String bagName) {
-		String bytes = Utils.getElementAsString(Utils.getElement(profileInfo, bagName + ".data"), "Hz8IAAAAAAAAAD9iYD9kYD9kAAMAPwI/Gw0AAAA=");
+		String bytes = Utils.getElementAsString(
+			Utils.getElement(profileInfo, bagName + ".data"),
+			"Hz8IAAAAAAAAAD9iYD9kYD9kAAMAPwI/Gw0AAAA="
```

Comment:
Btw, this string is still not valid NBT. But at this point i kind of want to keep that in as a meme.

Reply:
yeah I was wondering what it did, but IntelliJ formatted it, I wasn't trying to fix it